source/_includes/installation/operating_system.md

```diff
@@ -71,46 +76,96 @@ To boot Home Assistant OS, the BIOS needs to have UEFI boot mode enabled and Sec
 
 The BIOS configuration is now complete.
 
-Next, we need to write the Home Assistant Operating System image to the "boot medium", which is the medium your x86-64 hardware will boot from when it is running Home Assistant.
+## Write HAOS onto your x86-64 hardware
+
+Next, you need to write the Home Assistant Operating System image to the *boot medium*, which is the medium your x86-64 hardware will boot from when it is running Home Assistant.
 
 <div class='note'>
-HAOS has no integrated installer that writes the image automatically, you must write it manually using e.g. Etcher.
+HAOS has no integrated installer that writes the image automatically. You will write it manually using either the <b>Disks</b> utility from Ubuntu or Balena Etcher.
 </div>
 
-Typically an internal medium like S-ATA hard disk, S-ATA SSD, M.2 SSD, or a non-removable eMMC is used for the x86-64 boot medium. Alternatively, an external medium can be used such as a USB SDD, though this is not recommended.
+Typically, an internal medium like S-ATA hard disk, S-ATA SSD, M.2 SSD, or a non-removable eMMC is used for the x86-64 boot medium. Alternatively, an external medium can be used such as a USB SDD, though this is not recommended.
 
 To write the HAOS image to the boot medium on your x86-64 hardware, there are 2 different methods:
 
-1. Write the HAOS disk image from your desktop computer directly to the boot medium (e.g. using a USB to S-ATA adapter).
-If you can use this method, proceed to "[Write the image to your boot medium](#write-the-image-to-your-boot-medium)" and follow all steps. If you have non-removable internal mediums or don't have the necessary adapter, try the next method instead.
-
-2. Create a "live operating system" on a USB device running e.g. Ubuntu ([how-to guide](https://ubuntu.com/tutorials/try-ubuntu-before-you-install)). Insert it into your system and boot the live operating system. Then follow from step 2 in "[Write the image to your boot medium](#write-the-image-to-your-boot-medium)".
-
-{% details "Ubuntu dependencies for Etcher" %}
+  **Method 1**: Recommended method. It also works on laptops and PCs with internal hard disks. This method boots Ubuntu from a USB flash drive and installs the Home Assistant Operating System from there.
+
+  **Method 2**: With this method, you write the Home Assistant Operating disk image directly onto a boot medium. The steps are a bit more complex. If you have non-removable internal mediums (for example because you are using a laptop) or do not have the necessary adapter (for example an USB to eMMC adapter) use method 1 instead.
```

Comment:
I've not seen removable eMMC on x86-64 so far. A better example would be USB to S-ATA adapter here.

```suggestion
  **Method 2**: With this method, you write the Home Assistant Operating disk image directly onto a boot medium from your regular computer. The steps are a bit more complex. If you have non-removable internal mediums (for example because you are using a laptop) or do not have the necessary adapter (for example an USB to S-ATA adapter) use method 1 instead.
```